nixos/modules/virtualisation/google-compute-config.nix

```diff
@@ -69,6 +69,69 @@ in
   # GC has 1460 MTU
   networking.interfaces.eth0.mtu = 1460;
 
+  systemd.services.fetch-ssh-keys = {
```

Comment:
This should probably be moved into a module (like oslogin is), so it can be reused from people using something else than `google-compute-config.nix` as a base configuration.

We might also want to make it configurable whether we want to only download host keys or also ssh authorized keys.

Reply:
I think for now both are needed for NixOPS.

Reply:
Then NixOps itself / people making use of that feature in NixOps could set it to true, but Google really promotes using oslogin over keys in metadata: https://cloud.google.com/compute/docs/instances/access-overview#managing_user_access

Reply:
@flokli yeah I think it would be good to support oslogin, I'm not sure how much work in NixOPS is needed for that but it could be an improvement that we add later.
For now, I think it would be good to revive the image first to be able to release NixOPS 2.0. And then we should work on making oslogin work as well. But I agree for the images that will be used by NixOPS we could set this to true.

Reply:
Yeah, my initial idea was that this would only be temporarily reintroduced and that we would move to oslogin when that's supported by the plugin, but since this will likely still be needed for host keys, maybe it should be cleaned up a bit. I'll look into making it a separate module instead.

Reply:
After thinking more about this, I don't think a separate module would be a good fit - it's very specific to NixOps and probably wouldn't be used anywhere else. If we don't want to litter this to run in the normal gce image, we should probably create a NixOps specific one.

Reply:
In EC2 there is no difference, there is a single image and it does handle authorized_keys/host keys (https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/virtualisation/ec2-data.nix#L51), I think we just need to do the same here to make things simple.

Reply:
Then I would say this is done for now :)